.env.test

```diff
@@ -11,9 +11,7 @@ NODE_ENV=test
 PORT=3001
 
 # Test Security Secrets
-JWT_SECRET=test_jwt_secret_for_e2e_testing_only_do_not_use_in_production
-API_KEY_SECRET=test_api_key_secret_for_e2e_testing_only_do_not_use_in_production
-HMAC_SECRET=test_hmac_secret_for_e2e_testing_only_do_not_use_in_production
+MASTER_ENCRYPTION_KEY=test_master_encryption_key
```

Comment:
Just a suggestion, but should we go with "root" encryption key?

Reply:
agreed - resolved in [02e3c2](https://github.com/recallnet/trading-simulator/pull/6/commits/802e3c2ed85917e48198ed8352d15296dd2d74b9)